DualityEditorPlugins/EditorBase/PropertyEditors/Resources/FontContentPropertyEditor.cs

```diff
@@ -27,7 +27,7 @@ protected override void OnGetValue()
 			Font font = this.GetValue().FirstOrDefault() as Font;
 
 			bool lastCanRenderFont = this.canRenderFont;
-			this.canRenderFont = (font != null && font.EmbeddedTrueTypeFont != null);
+			this.canRenderFont = (font != null && font.GetGlyphBitmap(' ') != null);

```

Comment:
Ok - although this specific property canRenderFont is used in FontPropertyEditor.BeforeAutoCreateEditors to make all the properties of the font readonly. Can you suggest another way to fix this?